src/policy/fees.cpp

```diff
@@ -312,7 +312,12 @@ CBlockPolicyEstimator::CBlockPolicyEstimator()
         vfeelist.push_back(bucketBoundary);
     }
     vfeelist.push_back(INF_FEERATE);
-    feeStats.Initialize(vfeelist, MAX_BLOCK_CONFIRMS, DEFAULT_DECAY);
+    feeStats = new TxConfirmStats(vfeelist, MAX_BLOCK_CONFIRMS, DEFAULT_DECAY);
```

Comment:
heh... i agree, but i wanted to move code to .cpp file and then i was screwed by lack of a destructor.  I'm sure there is a better way, but I went around in circles chasing my tail for a while..  Maybe we could save that for an actual programmer to clean up later.